src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/Interfaces/ITranslationLayerRequestSender.cs

```diff
@@ -5,6 +5,7 @@ namespace Microsoft.TestPlatform.VsTestConsole.TranslationLayer.Interfaces
 {
     using System;
     using System.Collections.Generic;
+    using System.Threading.Tasks;
```

Comment:
If we do that, then the public non-async methods on VsTestConsoleWrapper would just end up calling the async methods internally and waiting them. Same end result, but with extra pointless overhead in the internal use of Tasks.